### PR TITLE
Moved a few options

### PR DESCRIPTION
**Pull request by julien51**
All in the title!

### CODE REVIEW

File: docs/tools/checkout/configuration.md [hunk 1]
```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
 
 The `paywallConfig` is a JSON object which includes a set of customizations for your experience. It includes the following elements:
 
-- `locks` : _required object_, a list of lock objects \(see below\).
+- `locks` : _required object_, a list of lock objects ([see below](#locks)).
 - `title`: _optional string_, a title for your checkout. This will show up on the head.
 - `icon`: _optional string_, the URL for a icon to display in the top left corner of the modal.
-- `callToAction`: _optional object_, a list of messages shown based on the state of the checkout modal \(see below\).
```
Comment (julien51) [inline docs/tools/checkout/configuration.md:41]: Deprecated!

File: docs/tools/checkout/configuration.md [hunk 2]
```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
...
-- `metadataInputs`: _optional array_, a set of input fields [as explained there](./collecting-metadata.md).
```
Comment (julien51) [inline docs/tools/checkout/configuration.md:42]: Better if it is _inside_ the lock object.

File: docs/tools/checkout/configuration.md [hunk 3]
```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
...
-- `persistentCheckout`: _optional boolean_: `true` \_\_if the modal cannot be closed, defaults to `false` when embedded. When closed, the user will be redirected to the `redirect` query param when using a purchase address \(see above\).
-- `useDelegatedProvider`: _optional boolean._ To be announced.
```
Comment (julien51) [inline docs/tools/checkout/configuration.md:44]: Never announced, we might re-introduce later.

File: docs/tools/checkout/configuration.md [hunk 4]
```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
...
-- `network`: _optional integer._ defaults to `1`. See below.
```
Comment (julien51) [inline docs/tools/checkout/configuration.md:45]: Inside the lock object

File: docs/tools/checkout/configuration.md [hunk 5]
```diff
@@ -35,21 +35,14 @@ This URL will redirect members to this page [`https://ouvre-boite.com/`](https:/
...
-- `referrer`: _optional string_. The address which will [receive UDT tokens](../../governance/the-unlock-token/) \(if the transaction is applicable\)
-- `messageToSign`: _optional string_. If supplied, the user is prompted to sign this message using their wallet. If using a checkout URL, a `signature` query param is then appended to the `redirectUri` \(see above\). If using the embedded paywall, the `unlockProtocol.authenticated` includes the `signature` attribute.
-- `pessimistic`: _optional boolean_ defaults to `false`_._ By default, to reduce friction, we do not require users to wait for the transaction to be mined before offering them to be redirected. By setting this to `true`, users will need to wait for the transaction to have been mined in order to proceed to the next step.
```
Comment (julien51) [inline docs/tools/checkout/configuration.md:48]: All elements below moved inside the lock!

Comment (searchableguy) [reply docs/tools/checkout/configuration.md:48]: I don't think we handle pessimistic on lock basis.

**Review by searchableguy — APPROVED** [review]
LGTM

**Pull request merged**